website/views/shared/profiles/achievements.jade

```diff
@@ -146,8 +146,8 @@ div(ng-if=':: profile.achievements.originalUser')
   
 div(ng-if='profile.achievements.challenges || user._id == profile._id')
   // This is a very strange icon to use. revisit
-  .achievement.achievement-karaoke(ng-show='profile.achievements.challenges')
-  div(ng-class='{muted: !profile.achievements.challenges}')
+  .achievement.achievement-karaoke(ng-show='profile.achievements.challenges.length')
```

Comment:
Sure, but most of the other achievements use `ng-show` on their achievement icons. If I'm going to change one, I should just go ahead and change them all, right?

Also, just to increase my understanding, why do you prefer `ng-if` over `ng-show` if they have the same result?